src/test/scala/uk/gov/hmrc/ct/computations/CP281bSpec.scala

```diff
@@ -24,44 +24,44 @@ import uk.gov.hmrc.ct.BoxValidationFixture
 import uk.gov.hmrc.ct.box.CtValidation
 import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
 
-class CP281bSpec extends WordSpec with Matchers with MockitoSugar with BoxValidationFixture[ComputationsBoxRetriever] {
+class CP281bSpec extends WordSpec with Matchers with MockitoSugar {
 
-  override val boxRetriever = makeBoxRetriever()
+//  override val boxRetriever = makeBoxRetriever()
```

Comment:
another commented code

Reply:
Removed it because they removed the validation from the box CP281b and changed to be the pre-populated field.